packages/react-native/React/Fabric/RCTConversions.h

```diff
@@ -34,6 +36,43 @@ inline std::string RCTStringFromNSString(NSString *string)
   return std::string{string.UTF8String ?: ""};
 }
 
+inline UIColor *_Nullable RCTUIColorFromDynamicColor(const facebook::react::Color &color)
```

Comment:
Could we have the PlatformColor struct hold on to a UIColor directly instead (eg using ManagedObjectWrapper)? That would save us trying to represent all of Apple's behaviour in our Cxx layer.

Reply:
@javache I made a wrapper to hold `UIColor`, please review it.

Reply:
Awesome work!